gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,33 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm = PbmService.new(server, username, password) if @apiVersion >= '5.5'
+  end
+
+  def pbmProfiles
```

Comment:
Yeah what I want to have work is for the refresher to be able to call it just like the others, e.g.: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/vmware/infra_manager/refresher.rb#L77
But I guess we don't need both methods for that if no one is going to call just pbmProfiles, could have it just be pbmProfilesByMor/ByUid.